journal.py

```diff
@@ -72,9 +73,51 @@ def list_view(request):
     return {'entries': entries}
 
 
+@view_config(route_name='detail', renderer='templates/detail.jinja2')
+def detail_view(request, session=None):
+    entry_id = request.params.get('id')
+    if session is None:
+        session = DBSession
+    entry = session.query(Entry).get(entry_id)
+    entry.content_md = markdown.markdown(
+        entry.content,
+        extensions=['markdown.extensions.codehilite'],
+        extension_configs={'markdown.extensions.codehilite': {'noclasses': True}}
+    )
```

Comment:
Interesting, so you are adding an extra attribute to your entry sort of on-the-fly.  What if your entry had a parameter like this instead:

```python
class Entry(Base):
    #...
    @property
    def content_md(self):
        return markdown.markdown(self.text, .......)
```
Would that simplify things at all?